image-configure.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This script is to configure the SD card on the Raspberry Pi
+
+set -e
+
+host=raspberrypi.local
+userHost=pi@$host
+
+echo "Installing saltstack"
+scp install-saltstack.sh $userHost:
+ssh $userHost "sudo ./install-saltstack.sh"
+ssh $userHost "sudo rm ./install-saltstack.sh"
+
+echo "Applying salt state"
+wget https://github.com/TheCacophonyProject/saltops/archive/prod.zip
+unzip prod.zip
+cd saltops-prod/
+sed -i '/maybe-reboot/d' salt/top.sls # We don't want to reboot when making the image
```

Comment:
How does top.sls get there if you're only downloading the prod branch? top.sls is only in master right?

Reply:
We maintain two `top.sls` files now. One in the Master branch and one in the other branches. This was done when `state-apply-test.sh` was added.

https://github.com/TheCacophonyProject/saltops/blob/prod/salt/top.sls

Reply:
ah yes... I forgot about that